tutorials/boundary_conditions.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>Please explain that bc_type_-methods are only defined for fluxes. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1073/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>To set a boundary condition do the following: </p><p>1 ...... bc_type_"flux_name" &lt;- ((Needs more explanation. Somewhere.)) Point is that we do not have a bc_type_pressure, which might be confusing if it is not mentioned. </p><p>2 ...... bc_values_"physical_quantity"</p><p> </p><p>bc_type_mechanics -&gt; bc_type_stress, keep the old one and throw an error (as help to users such that they rename it). Will be separate PR.</p>